EventHighway.Core/Services/Orchestrations/Events/V2/EventV2OrchestrationService.cs

```diff
@@ -16,19 +17,34 @@ namespace EventHighway.Core.Services.Orchestrations.Events.V2
     internal partial class EventV2OrchestrationService : IEventV2OrchestrationService
     {
         private readonly IEventV2ProcessingService eventV2ProcessingService;
+        private readonly IEventAddressV2ProcessingService eventAddressV2ProcessingService;
         private readonly IEventCallV2ProcessingService eventCallV2ProcessingService;
         private readonly ILoggingBroker loggingBroker;
 
         public EventV2OrchestrationService(
             IEventV2ProcessingService eventV2ProcessingService,
+            IEventAddressV2ProcessingService eventAddressV2ProcessingService,
             IEventCallV2ProcessingService eventCallV2ProcessingService,
             ILoggingBroker loggingBroker)
         {
             this.eventV2ProcessingService = eventV2ProcessingService;
+            this.eventAddressV2ProcessingService = eventAddressV2ProcessingService;
             this.eventCallV2ProcessingService = eventCallV2ProcessingService;
             this.loggingBroker = loggingBroker;
         }
 
+        public ValueTask<EventV2> SubmitEventV2Async(EventV2 eventV2) =>
+        TryCatch(async () =>
+        {
+            ValidateEventV2IsNotNull(eventV2);
+
+            _ = await this.eventAddressV2ProcessingService
```

Comment:
```suggestion
            await this.eventAddressV2ProcessingService
```

Do we really need  this  `_ =`

Reply:
Yeah, this shows as the method is not just ValueTask but that it has a value in it. And it make it clear for us that we calling this method for a special case like: validation  or exception

Reply:
@ZafarUrakov Did we have a discussion about that before?
Because in some services we are just calling  the methods and not using any variable.
And it looks strange to me. It is first time I have seen this pattern